platform/assets/js/app.js

```diff
@@ -66,6 +66,15 @@ Hooks.ScrollToTop = {
     }
 }
 Hooks.InfiniteScroll = InfiniteScroll;
+Hooks.PasteFromClipboard = {
+    mounted() {
+        this.el.addEventListener("paste", e => {
+            pastedFiles = e.clipboardData.files;
+            console.log(pastedFiles)
+            this.pushEvent("paste", { pastedFiles });
+        })
+    }
+}
 
 // Used by the pagination button to scroll back up to the top of the page.
 window.scrollToTop = () => {
```

Comment:
Be sure to add the attachments to your `.gitignore` so that they are not included in the PR.